libs/aws/langchain_aws/retrievers/bedrock.py

```diff
@@ -123,11 +156,16 @@ def _filter_by_score_confidence(self, docs: List[Document]) -> List[Document]:
         return filtered_docs
 
     def _get_relevant_documents(
-        self, query: str, *, run_manager: CallbackManagerForRetrieverRun
+        self,
+        query: str,
+        *,
+        run_manager: CallbackManagerForRetrieverRun,
+        next_token: Optional[str] = None,
     ) -> List[Document]:
         response = self.client.retrieve(
             retrievalQuery={"text": query.strip()},
             knowledgeBaseId=self.knowledge_base_id,
+            nextToken=next_token,
```

Comment:
Assuming nextToken is required field, otherwise let's not add this.
```suggestion
            nextToken=None,
```

Reply:
I put this at None as I removed the argument nextToken.